configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -546,25 +546,6 @@
         </div>
     </section>
 
-
```

Comment:
There is nothing here, as far as I can tell, so no need to preserve this logic.  Doing a search across the codebase for "mh-epilepsy-section" shows a section with this id in "epilepsy.xml", but not in this form.  If/when we want to add some conditional section in for epilepsy, we can follow the new patterns.